CONTRIBUTING.md

```diff
@@ -116,23 +116,35 @@ and the final version are only comitted and tagged in this branch. The master br
 development versions, allowing us to keep the branches completely separate without merging releases
 back into master.
 
-The exact steps for an exemplary `1.2.3` release might look like this:
- 1. Initially, the version on the latest master is `1.2.3-dev`
- 2. A new `v1.2.3` branch is created for the release
- 3. On master, the version is bumped to `1.2.4-dev`
- 4. In the branch, the version is bumped to `1.2.3-rc1`
- 5. The new commit in the branch is tagged as `v1.2.3-rc1`
- 6. A GitHub release is created for the `v1.2.3-rc1` tag
- 7. The changelog since the last release (stable or RC)
-        is added to the GitHub release description
- 8. Bug fixes are cherry-picked from master into the branch and steps 4-7
-        are repeated until no major issues are found in the release candidates
- 9. In the branch, the version is bumped to `1.2.3`
- 10. The new commit in the branch is tagged as `v1.2.3`
- 11. A GitHub release is created for the `v1.2.3` tag
- 12. The changelog since the last stable release (**not** RC)
-        is added to the GitHub release description
- 13. The `-dev` is stripped from previous changelog entries
+The exact steps for an exemplary `0.2.0` release might look like this:
+  1. Initially, the version on the latest master is `0.2.0-dev`
+  2. A new `v0.2.0` branch is created for the release
+  3. On master, the version is bumped to `0.3.0-dev`
+  4. In the branch, the version is bumped to `0.2.0-rc1`
+  5. The new commit in the branch is tagged as `v0.2.0-rc1`
+  6. A GitHub release is created for the `v0.2.0-rc1` tag
+  7. The changelog since the last release (stable or RC) is added to the GitHub release description
+  8. Bug fixes are cherry-picked from master into the branch and steps 4-7 are repeated until no
+     major issues are found in the release candidates
+  9. In the branch, the version is bumped to `0.2.0`
+ 10. The new commit in the branch is tagged as `v0.2.0`
+ 11. A GitHub release is created for the `v0.2.0` tag
+ 12. The changelog since the last stable release (**not** RC) is added to the GitHub release
+     description
+ 13. The `-dev` is stripped from the `0.2.0-dev` changelog entries on master
+
+On the master and with new planned releases, the patch version is never bumped, but only the minor
+is increased. This allows for creating releases to fix bugs which build on previous minor releases
+and just increment the patch version, without having to decrement or advance the version based on
+these unplanned releases.
+
+The exact steps for an exemplary `0.2.3` release might look like this:
+ 1. Initially, the version on the latest master is `0.3.0-dev` and the latest release was `0.2.2`
+ 2. A new `v0.2.3` branch is forked from the `v0.2.2` branch
+ 4. All bug fixes are cherry-picked from master into the `v0.2.3` branch
+ 5. The version is bumped to `v0.2.3-rc1` and the changelog's `0.3.0` entries are moved to `0.2.3`
+ 6. Steps 5-12 for regular releases are identical
```

Comment:
This seems a bit odd. I don't get the `Steps 5-12`, it's a different list with its own `5` tbw.